src/mudlet-lua/lua/Other.lua

```diff
@@ -3,7 +3,7 @@
 ----------------------------------------------------------------------------------
 
 mudlet = mudlet or {}
-mudlet.supportscoroutines
+mudlet.supportscoroutines = true
```

Comment:
Or is it that that will get stored when the profile is saved and then it will be present when an older Mudlet version loads the code... 😕